### PR TITLE
Git machete tab on git init + drop support for 2020.1 and 2020.2

### PR DESCRIPTION
**Pull request by mkondratek**
Probably I did not fix all the problems but at least we are consistent with git4idea VCS log 😆 

Screenshot 2022-05-12 at 16 27 04

### CODE REVIEW

File: frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMacheteVisibilityPredicate.java [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 @CustomLog
 public class GitMacheteVisibilityPredicate implements NotNullFunction<Project, Boolean> {
 
   @Override
   public Boolean fun(Project project) {
-    // `com.intellij.openapi.vcs.ProjectLevelVcsManager#checkVcsIsActive(String)` returns `true` when the specified
-    // VCS is used by at least one module in the project. Therefore it is guaranteed that Git Machete plugin tab
-    // is created only when a git repository exists.
-    boolean predicateResult = ProjectLevelVcsManager.getInstance(project).checkVcsIsActive(GitVcs.NAME);
+    boolean predicateResult = !VcsProjectLog.getLogProviders(project).isEmpty();
```
[the remark this answers is not in the document]
Comment (mkondratek) [reply frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMacheteVisibilityPredicate.java:13]: nope - that was a problem with our sight abilities XD the `isStagingAreaAvailable` referred to something different than VCS log. now it is a proper predicate

[the remark this answers is not in the document]
Comment (PawelLipski) [reply frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/root/GitMacheteVisibilityPredicate.java:13]: Whoops conjunctivitis/keratitis have far-reaching consequences indeed 😅

**Review by PawelLipski — APPROVED** [review]

**Comment by PawelLipski** [comment]
Huh compat problems on 2020.1 and 2020.2... maybe that's the moment to ditch them and bump plugin version to 2.0.0 🤔

File: src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala [hunk 2]
```diff
@@ -63,11 +63,7 @@ object UITestSuite
       |  }
       |
       |  resolvers.jbr.repositories = [
-      |    official,
-      |
-      |    # Let's fall back to 2020.2's JBR for 2020.1's builds.
```
Comment (mkondratek) [inline src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala:68]: @PawelLipski is it ok?

Comment (PawelLipski) [reply src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala:68]: :yes:

File: version.gradle [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ext {
-  PROSPECTIVE_RELEASE_VERSION = '1.4.0'
+  PROSPECTIVE_RELEASE_VERSION = '2.0.0'
```
Comment (PawelLipski) [inline version.gradle:2]: Pls mention the discontinuation of old IDEA versions in change notes

File: CHANGE-NOTES.html [hunk 4]
```diff
@@ -1,4 +1 @@
-<p>Fix docs on the shortcut that opens Git Machete Tab <i>(command option shift M)</i> on Mac.</p>
-
-<p><i>Merge Parent Into Current</i> action has been added.
-    The action merges a parent branch into the current branch, as suggested by @mkondratek and @pedroadame.</p>
+<p>Dropped support for IntelliJ 2020.2.</p>
```
Comment (PawelLipski) [inline CHANGE-NOTES.html:1]: The other entries are also to be retained! As we'll probably just release v2.0.0, skipping v1.4.0 entirely

File: CHANGE-NOTES.html [hunk 5]
```diff
@@ -2,3 +2,5 @@
 
 <p><i>Merge Parent Into Current</i> action has been added.
     The action merges a parent branch into the current branch, as suggested by @mkondratek and @pedroadame.</p>
+
+<p>Dropped support for IntelliJ 2020.2.</p>
```
Comment (PawelLipski) [inline CHANGE-NOTES.html:6]: 🤓 
```suggestion
<p>Dropped support for IntelliJ 2020.1 and 2020.2.</p>
```

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**